package.json

```diff
@@ -5,22 +5,26 @@
   "main": "lib/main.js",
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --silent",
+    "test-coverage": "jest --silent --coverage"
```

Comment:
Why `--silent`?

Reply:
If `--silent` is not used, all the debug and console logs get printed along with the test results.

Reply:
That's okay right? We need something to debug a failed test?